web/src/main/java/ua/com/vertex/controllers/UserDetailsController.java

```diff
@@ -56,30 +56,20 @@ public ModelAndView saveUserData(
             @RequestPart(value = "imagePassportScan", required = false) MultipartFile imagePassportScan,
             @RequestPart(value = "imagePhoto", required = false) MultipartFile imagePhoto,
             @Valid @ModelAttribute(USERDATA_MODEL) User user,
-            BindingResult bindingResult, ModelAndView modelAndView) {
+            BindingResult bindingResult, ModelAndView modelAndView) throws IOException {
 
         modelAndView.setViewName(PAGE_JSP);
         if (bindingResult.hasErrors()) {
-            modelAndView.addObject("msg", "WARNING!!! User data is updated, but not saved");
+            modelAndView.addObject("msg",
+                    "WARNING!!! User data is updated, but not saved. One or more fields are invalid");
```

Comment:
"WARNING!!!' sound like you are from seals or other military organizations. Please can you change the message to something like "We'd like to save your changes but we can't while one or more fields are invalid"

Reply:
Not my message. It was merged long before, without objections, so why would I change it?

Reply:
Well, my fault that i've missed that before. can you please change it?